src/aleph/http.clj

```diff
@@ -186,7 +192,7 @@
                                     client/close-connection))
                :control-period control-period
                :max-queue-size max-queue-size
-               :controller (Pools/utilizationController
+               :controller (create-pool-ctrl-fn
                             target-utilization
                             connections-per-host
                             total-connections)
```

Comment:
- This setup doesn't support using Dirigiste's fixed pools (or even other utilization pools), because the arities don't match. We should change it to a 0-arity thunk up above.